model/post.go

```diff
@@ -280,6 +280,8 @@ type PostCountOptions struct {
 	MustHaveFile    bool
 	MustHaveHashtag bool
 	ExcludeDeleted  bool
+	UsersPostsOnly  bool
+	AllowFromCache  bool
```

Comment:
Same as below, can we add a comment here that says `AllowCache` only works if `ExcludeDeleted` and `UserPostsOnly` are unset?